recipes/perl-class-accessor/recipe.yaml

```
@@ -0,0 +1,48 @@
+schema_version: 1
+
+context:
+  name: perl-class-accessor
+  version: 0.51
+  sha256: bf12a3e5de5a2c6e8a447b364f4f5a050bf74624c56e315022ae7992ff2f411c
+
+package:
+  name: ${{ name }}
+  version: ${{ version }}
+
+source:
+  url: https://cpan.metacpan.org/authors/id/K/KA/KASEI/Class-Accessor-${{ version }}.tar.gz
+  sha256: ${{ sha256 }}
+
+build:
+  number: 0
+  noarch: generic
+  script:
+    - perl Makefile.PL INSTALLDIRS=site NO_PERLLOCAL=1 NO_PACKLIST=1
+    - make
+    - make test
+    - make install
+
+requirements:
+  build:
+    - make
+  host:
+    - perl=5.32
+  run:
+    - perl>=5.32
+
+tests:
+  - perl:
+      uses:
+        - Class::Accessor
+        - Class::Accessor::Fast
+        - Class::Accessor::Faster
+
+about:
+  license: perl_5
```

```suggestion
  license: Artistic-1.0 OR GPL-1.0-or-later
  license_file:
    - perlartistic.1
    - perlgpl.1
```
This is the proper license metadata for the `perl_5` licenses.  Before using this, we need to check for each module in the "Browse" section in CPAN if it includes its own `LICENSE` file, which should take priority over the generic `perl_5` license.  For reasons I don't fully understand, a significant fraction of CPAN modules incorrectly leave the License field in the metadata as `perl_5` despite providing their own `LICENSE` file.